### ISSUE TITLE
Can we measure the difference of 2 gray scale images?

### ISSUE THREAD

**Issue by leviethung2103** [richzhang/PerceptualSimilarity#29]
Hello author, 

Can we measure the difference of 2 gray scale images? Or this metric is only used for RGB images? 

Thank you.

**Comment by leviethung2103** [comment]
I've run the model with my two gray scales. I am not sure that the value given from network is correct.  

Furtheremore, the lower LPIPS is, the better image is. Am I correct. Thank you

**Comment by richzhang** [comment]
Yes, the lower the better. We didn't verify on perceptual judgments with grayscale images only, but I think it is reasonable to expect consistent results to color images.

**Comment by leviethung2103** [comment]
Thank you so much.

**Comment by cyrilzakka** [comment]
My apologies for re-opening this thread after such a long while - is it possible to obtain a greyscale version of vgg16_perceptual? I trained a styleGAN model on a greyscale dataset. Currently the projector uses this model to project images in latent space. Unfortunately when running the projector I get:
`Dimension 1 in both shapes must be equal, but are 1 and 3. Shapes are [1,1,128,128] and [?,3,?,?]`

If not, I have to retrain the GAN which I'm leaving as the last option.

**Comment by richzhang** [comment]
I think you can convert the images into 3 channels using [expand](https://pytorch.org/docs/stable/tensors.html#torch.Tensor.expand)
